astropy/coordinates/builtin_frames/baseradec.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from ... import units as u
+from .. import representation as r
+from ..baseframe import BaseCoordinateFrame, RepresentationMapping
+
+__all__ = ['BaseRADecFrame']
+
+_base_radec_docstring = """Parameters
+    ----------
+    representation : `BaseRepresentation` or None
+        A representation object or ``None`` to have no data (or use the other
+        keywords below).
+
+    ra : `Angle`, optional, must be keyword
```

Comment:
All of these instances of "must be keyword" are from master...would you like me to address them here?
e.g., https://github.com/astropy/astropy/blob/master/astropy/coordinates/builtin_frames/icrs.py#L27